scripts/setupvars/setupvars.sh

```diff
@@ -51,7 +51,7 @@ if [ -e "$INSTALLDIR/runtime/3rdparty/tbb" ]; then
     if [[ "$OSTYPE" == "darwin"* ]]; then
         export DYLD_LIBRARY_PATH=$INSTALLDIR/runtime/3rdparty/tbb/lib:${DYLD_LIBRARY_PATH:+:DYLD_LIBRARY_PATH}
     fi
-    export LD_LIBRARY_PATH=$INSTALLDIR/runtime/3rdparty/tbb/lib:${LD_LIBRARY_PATH:+:$LD_LIBRARY_PATH}
+    export LD_LIBRARY_PATH=$INSTALLDIR/runtime/3rdparty/tbb/lib/intel64/gcc4.7:${LD_LIBRARY_PATH:+:$LD_LIBRARY_PATH}
```

Comment:
cherry-picking this from the https://github.com/openvinotoolkit/openvino/pull/9240is likely not needed for this PR, as this is onetTBB specific path change